src/main/java/io/voucherify/client/model/async_actions/AsyncActionResponse.java

```diff
@@ -0,0 +1,34 @@
+package io.voucherify.client.model.async_actions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.voucherify.client.model.common.response.applicableTo.ApplicableToResponse;
+import io.voucherify.client.model.validationRules.response.BusinessValidationRuleAssignmentList;
+import io.voucherify.client.model.voucher.Discount;
+import io.voucherify.client.model.voucher.Gift;
+import io.voucherify.client.model.voucher.LoyaltyCard;
+import io.voucherify.client.model.voucher.VoucherType;
+import io.voucherify.client.model.voucher.response.VoucherPublishResponse;
+import io.voucherify.client.model.voucher.response.VoucherRedemptionResponse;
+import lombok.*;
+
+import java.util.Date;
+import java.util.Map;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
czy to jest akceptowalne?
tylko pod testy to zmienilem, np. Voucher ma osobną klase uzytą w testach, co tez jest średnie bo to duplikacja kodu

Reply:
jest akceptowalne, jak ktoś sobie sam stworzy response'a to jego sprawa 🤷